slides/07.Rmd

```diff
@@ -62,3 +72,7 @@ title: Getting Started with the Cloud
 ### Cohort 1
 
 `r knitr::include_url("https://www.youtube.com/embed/O6lhqSz-8Dw")`
+
+## Cohort 2
+
+`r knitr::include_url("https://www.youtube.com/embed/yk8LqHzlS7s")`
```

Comment:
```suggestion
```

Can you remove this videos section (and the `## Meeting Videos` part above it, but GitHub didn't let me select back that far), and create `/videos/07/02.qmd` from `/videos/07/01.qmd` (with `yk8LqHzlS7s` instead of `O6lhqSz-8Dw`)? I thought I'd removed this section from everything, I'll need to go back through and make sure, but it'd be great to get in the habit of doing it "right" (the new definition of "right", that is)!